airflow/config_templates/config.yml

```diff
@@ -913,6 +913,18 @@ logging:
       type: boolean
       example: ~
       default: "False"
+    enable_arbitrary_log_shipping:
```

Comment:
I don't think "arbitrary" is a good word to use. Maybe `enable_global_task_logging` or something similar?

We've used arbitrary when talking about this functionality since what we log doesn't actually matter (specifically the content of the log messages and where it comes from), but I don't think it's a good way to present the feature to users.

Reply:
I agree. how about the word "ancillary" or the synonyms of ancillary like "auxiliary" / "supplementary" instead of arbitrary? 

cc: @dstandish

Reply:
maybe it would suffice to call it `task_log_shipper`?

task log cus, we are shipping to task logs

ship because, we're sending it from a place more removed from the task itself.

Reply:
so here it would be called `enable_task_log_shipper`